docs/_providers/axfrddns.md

```diff
@@ -0,0 +1,130 @@
+---
+name: AXFRDDNS
+title: AXFR+DDNS Provider
+layout: default
+jsId: AXFRDDNS
+---
+# AXFR+DDNS Provider
+
+This provider is able to work with any authoritative DNS server
+accepting AXFR requests (RFC5936) and Dynamic Update (RFC2136).
+
+It has been tested with [BIND](https://www.isc.org/bind/),
+[Knot](https://www.knot-dns.cz/), and
+[Yadifa](https://www.yadifa.eu/home).
+
+## Configuration
+
+### Authentication
+
+The AXFR+DDNS provider might work without anything in `creds.json` if
+the primary master of the zone accepts transfers and updates without
+TSIG authentication. But for non widely-open server, the
+authentication keys should be provided in `creds.json`.
+
+{% highlight json %}
+{
+    "axfrddns": {
+        "transfer-key": "hmac-sha256:transfer-key-id:Base64EncodedSecret=",
+        "update-key": "hmac-sha256:update-key-id:AnotherSecret="
+    }
+}
+{% endhighlight %}
+
+The `transfer-key` will be used to authenticate AXFR request, and the
+`update-key` will be used to authenticate the Dynamic Updates. Both
+keys are optional, and you could provide only one them.
+
+If distinct zones requires distinct keys, you might instantiate the
+provider multiple times:
+
+{% highlight javascript %}
+var AXFRDDNS_A = NewDnsProvider('axfrddns-a', 'AXFRDDNS'}
+var AXFRDDNS_B = NewDnsProvider('axfrddns-b', 'AXFRDDNS'}
+{% endhighlight %}
+
+And update `creds.json` accordingly:
+
+{% highlight json %}
+{
+    "axfrddns-a": {
+        "transfer-key": "hmac-sha256:transfer-key-id:Base64EncodedSecret=",
+        "update-key": "hmac-sha256:update-key-id:AnotherSecret="
+    },
+    "axfrddns-b": {
+        "transfer-key": "hmac-sha512:transfer-key-id-B:SmallSecret=",
+        "update-key": "hmac-sha512:update-key-id-B:YetAnotherSecret="
+    }
+}
+{% endhighlight %}
+
+### Default nameservers
+
+The AXFR+DDNS provider can be configured with a list of default
+nameservers. They will be added to all the zones handled by the
+provider.
+
+This list can be provided either as metadata or in `creds.json`. Only
+the later allows `get-zones` to work properly.
+
+{% highlight javascript %}
+var AXFRDDNS = NewDnsProvider('axfrddns', 'AXFRDDNS',
+    'default_ns': [
+        'ns1.example.tld.',
+        'ns2.example.tld.',
+        'ns3.example.tld.',
+        'ns4.example.tld.'
+    ]
+}
+{% endhighlight %}
+
+{% highlight json %}
+{
+   nameservers = "ns1.example.tld,ns2.example.tld,ns3.example.tld,ns4.example.tld"
+}
+{% endhighlight %}
+
+### Primary master
+
+By default, the AXFR+DDNS provider will send the AXFR requests and the
+updates to the first nameserver of the zone, usually known as the
```

Comment:
This reads as AXFR (requests and updates) but I think you mean AXFR (requests) and DDNS (updates).  If that's true, add "TSIG" before the word "updates".

Reply:
Done, thanks.